posthog/models/dashboard_templates.py

```diff
@@ -5,13 +5,26 @@
 from posthog.models.utils import UUIDModel
 
 
+class DashboardTemplateManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().exclude(deleted=True)
+
+
 class DashboardTemplate(UUIDModel):
+    objects = DashboardTemplateManager()
+    objects_including_soft_deleted = models.Manager()
+
     team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE, null=True)
     template_name: models.CharField = models.CharField(max_length=400, null=True)
-    dashboard_description: models.CharField = models.CharField(max_length=400, null=True)
-    dashboard_filters: models.JSONField = models.JSONField(null=True)
+    dashboard_description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    dashboard_filters: models.JSONField = models.JSONField(null=True, blank=True)
     tiles: models.JSONField = models.JSONField(default=list)
-    tags: ArrayField = ArrayField(models.CharField(max_length=255), default=list)
+    variables: models.JSONField = models.JSONField(null=True, blank=True)
+    tags: ArrayField = ArrayField(models.CharField(max_length=255), default=list, blank=True)
+    created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True, blank=True)
+    created_by: models.ForeignKey = models.ForeignKey("User", on_delete=models.SET_NULL, null=True, blank=True)
+    deleted: models.BooleanField = models.BooleanField(default=False)
+    image_url: models.CharField = models.CharField(max_length=400, null=True, blank=True)
```

Comment:
as above note that 400 characters is probably too low for a URL, see comment below here in this file